vlib/compress/deflate/README.md

```diff
@@ -0,0 +1,21 @@
+## Description:
+
+`compress.deflate` is a module that assists in the compression and
+decompression of binary data using `deflate` compression
+
+NOTE: To decompress gzip, discard first 10 bytes of compressed bytes
+then use `compress.deflate.decompress`. (Header validation won't be
+performed in this case)
```

Comment:
This is wrong as gzip headers *can* be longer and gzip also has an 8 byte checksum at the end.

Reply:
maybe you can rewrite the description when your gzip implementation is done? I would be appreciate for this.

Reply:
I will! Currently writing tests for it and should be ready soon.